docs-website/sidebars.js

```diff
@@ -273,14 +273,60 @@ module.exports = {
         ],
       },
     ],
+    Authentication: [
+      {
+        type: "doc",
+        id: "docs/authentication/README",
+        label: "Overview",
+      },
+      {
+        type: "doc",
+        id: "docs/authentication/concepts",
+        label: "Concepts",
+      },
+      {
+        "Frontend Authentication": [
+          "docs/authentication/guides/jaas",
+          {
+            "OIDC Authentication": [
+              "docs/authentication/guides/sso/configure-oidc-react",
+              "docs/authentication/guides/sso/configure-oidc-react-google",
+              "docs/authentication/guides/sso/configure-oidc-react-okta",
+              "docs/authentication/guides/sso/configure-oidc-react-azure",
+            ],
+          },
+          "docs/authentication/guides/add-users",
+        ],
+      },
+      {
+        type: "doc",
+        id: "docs/authentication/introducing-metadata-service-authentication",
+        label: "Metadata Service Authentication",
+      },
+      {
+        type: "doc",
+        id: "docs/authentication/personal-access-tokens",
+        label: "Personal Access Tokens",
+      },
+    ],
+    Authorization: [
+      {
+        type: "doc",
+        id: "docs/authorization/README",
+        label: "Introduction",
```

Comment:
It will appear as Overview. 
I'm not sure what label is supposed to be, but what gets rendered is the title of the document (docs/authorization/README.md) itself.